docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.1.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v2.0.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v2.0.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "2.0.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v2.0.0"
+    },
+    {
+      "name": "coastal.v1.0.1"
+    },
+    {
+      "name": "coastal.v1.1.0"
+    },
+    {
+      "name": "coastal.v1.0.0"
+    },
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+Once again, we should see the `Catalog` update it's resolved reference and this time we expect that
+nothing happens to the `Operator` resource.
+
+Updating the `Operator`'s version range will still result in a resolution failure since the version installed
+is a lower major version than specified by the version range. Try it by running:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: coastal
+spec:
+  packageName: coastal
+  version: 2.0.x
+EOF
+```
+
+Watch for a resolution failure with:
+```sh
+kubectl get operator/coastal -o yaml -w
+```
+
+## To the escape hatch!
+To tell operator-controller to ignore the semver policies and allow upgrades across major versions,
+set the `Operator`'s `spec.upgradeConstraintPolicy` to `Ignore` with:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: coastal
+spec:
+  packageName: coastal
+  upgradeConstraintPolicy: Ignore
+  version: 2.0.x
+EOF
+```
+
+We should see that eventually the `Operator` will resolve and install the `v2.0.0` bundle we added to the
+catalog image in the previous step. Watch this happen with:
+```sh
+kubectl get operator/coastal -o yaml -w
+```
+
+## Attempting to downgrade by changing the version range
+When the `Operator`'s `spec.UpgradeConstraintPolicy` is set to `Enforce`, downgrades are disabled. 
```

Comment:
```suggestion
When the `Operator` resource's `spec.UpgradeConstraintPolicy` is set to `Enforce`, downgrades are disabled. 
```

Reply:
Consider:
"We can disable downgrades by setting the `Operator` resource's `spec.UpgradeConstraintPolicy` field to `Enforce`.

Reply:
[08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)